tools/makeflash.sh

```diff
@@ -3,7 +3,9 @@
 #
 #      ./makeflash.sh [-C size] <input dir> <output.img> [partitions]
 #
-MKFLASH_TAG="$(linuxkit pkg show-tag pkg/mkimage-raw-efi)"
+EVE="$(cd "$(dirname "$0")" && pwd)/../"
+PATH="$EVE/build-tools/bin:$PATH"
+MKFLASH_TAG="$(linuxkit pkg show-tag "$EVE/pkg/mkimage-raw-efi")"
```

Comment:
Lot of repetition between the various `.sh` files. Any chance of dedup? Maybe a `functions.sh` or some single other script that would provide this?

Reply:
Great mind think alike ;-) Here's the rub: in order to pick up anything like functions.sh the script will have to find itself. Thats' the first two lines right there.

The rest of it -- I agree -- needs to be centralized, but I was thinking replacing all these individual scripts with a single one and having symlinks pointing to it (and hence doing resolution of what needs to be done depending on the name the script is called by).

That, however, will be a different PR. But even so -- what do you think about this idea of one script instead of many.